src/DifferenceEquations.jl

```diff
@@ -6,34 +6,30 @@ using DistributionsAD
 using LinearAlgebra
 using CommonSolve
 using Zygote
+using LoopVectorization
+using Parameters
```

Comment:
Did you add this for `@unpack`? `@unpack` is defined in the more lightweight UnPack.jl (https://github.com/mauro3/UnPack.jl).

Reply:
Yes, thanks.  May clean that up on the next round.